sig-cli/annual-report-2021.md

```diff
@@ -0,0 +1,73 @@
+# SIG CLI Annual Report 2021
+
+This report reflects back on CY 2020 and was written in March 2021.
+
+## Operational
+
+* How are you doing with operational tasks in [sig-governance.md](https://github.com/kubernetes/community/blob/master/committee-steering/governance/sig-governance.md)?
+
+    * Is your README accurate? have a CONTRIBUTING.md file?
+        * Yes, [README.md](https://github.com/kubernetes/community/blob/master/sig-cli/README.md) is accurate and up-to-date.
+        * Our [CONTRIBUTING.md](https://github.com/kubernetes/community/blob/master/sig-cli/CONTRIBUTING.md) requires an update as a follow-up to our recent completion of the [move to staging](https://github.com/kubernetes/enhancements/issues/1020).
+
+    * All subprojects correctly mapped and listed in sigs.yaml?
+        * The current list is up-to-date, but this will soon require an update as soon as [KUI moves to kubernetes-sig](https://github.com/kubernetes/org/issues/2461).
```

Comment:
nit: this is done now :smile:

Reply:
Correct, but I assumed the report to be for anything before 1.21 release :wink: so I left it as open. 
@parispittman we should probably state clear timeline for a report, so that we know what to report as in-progress or what as completed.

Reply:
@soltysh i think its fine as is for reason enumerated